cvePipeline.py

```diff
@@ -0,0 +1,10 @@
+from cvereporter import fetch_vulnerabilities
+from cvereporter import report
+
+bom = report.get_base_bom()
+#todo: take date as arg or figure out other way to seed 
+vulns = fetch_vulnerabilities.fetch_cves('2023-01-17')
```

Comment:
I'd love to see the fetching of CVEs result in a serialized JSON file. That way we can study it to check this part of the pipeline (which is highly likely to be affected by external OpenJDK website changes) is working as expected.

The pipeline can continue, and eventually with the option of running from the serialized fetched vulnerabilities file - so we don't have to run the full pipeline each time, or we can run from a patched file, etc.

Reply:
this is now the case.